Server/Ebenezer/User.cpp

```diff
@@ -8677,6 +8677,20 @@ void CUser::InitType4()
 	m_bType4Flag = FALSE;
 }
 
+int CUser::GetNumberOfEmptySlots()
+{
+	int emptySlotsCount = 0;
+
+	for(int i = SLOT_MAX; i < SLOT_MAX + HAVE_MAX; i++)
```

Comment:
Select the method and run a quick formatter pass (press CTRL+K, then press F) over it.